_data/apps.yml

```diff
@@ -1101,3 +1101,14 @@
     - "database"
     - "design"
   icon: "collectie.png"
+-
+  name: "Calculator"
+  description: "A Simple Calculator with Google UI"
+  website: "https://softvilla.com.pk"
+  repository: "https://github.com/sabeehmurtaza9/Calculator/"
```

Comment:
The code of your application does not appear to be in this repository, is it supposed to be there?